tool/tctl/common/auth_command.go

```diff
@@ -757,6 +757,14 @@ client_encryption_options:
    keyfile: /path/to/{{.output}}.key
    truststore:  /path/to/{{.output}}.cas
    require_client_auth: True
+
+To enable x509 certificate authentication on you Scylla server (available from
+Scylla version 5.4.0), add the following to your scylla.yaml configuration file:
+
+authenticator: com.scylladb.auth.CertificateAuthenticator
+auth_certificate_role_queries:
+    - source: SUBJECT
+      query: CN=([^,\s]+)
```

Comment:
> However, since Scylla doesn't support multiple authentication methods, any access to the database would require the usage of Teleport-issued certificates (for example, through Machine ID). So, I'm not sure if this will be desired by default.

If true, could you add a warning here?